api-reference/openapi_spec.json

```diff
@@ -4345,7 +4345,7 @@
             "content": {
               "application/json": {
                 "schema": {
-                  "$ref": "#/components/schemas/PayoutListResponse"
+                  "$ref": "#/components/schemas/PayoutListResponseV2"
```

Comment:
Why is this included in the v1 OpenAPI specification? 

1. Is it intentional, or was it meant to be only updated in the v2 OpenAPI specification?
2. If it was intentional, can't we make changes in the `PayoutListResponse` struct instead?